docs/UserGuide.md

```diff
@@ -923,11 +920,60 @@ Command Words Accepted: `list`, `ls` (case-insensitive)
 > ```
 
 #### Clear Transactions
+
 Clears all transactions in UniCa$h.
 
 Command: `clear_transactions`
 
-Command Words Accepted: `clear_transactions` (case-insensitive)
+Command Words Accepted: `clear_transactions` only (case-insensitive)
+
+<div class="callout callout-info" markdown="span" style="margin-bottom: 20px;">
+This command irreversibly deletes all transactions in UniCa$h. Thus, as an added layer
+of safety, the command has no short form alternatives and is intentionally lengthy. Additionally,
+it cannot be followed with any arguments, options, or non-whitespace text.
+</div>
+
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+This command is subject to the filter applied by `find` or `get_total_expenditure`
```

Comment:
I don't think this is the case, even after applying a filter and then using clear, the final transaction list (after using `list` is blank)

Reply:
Yes, but if you add a transation after `clear` which doesn't match the filter, then it has the same issue

Reply:
Wait let me go and reconfirm this

Reply:
Ok, then I think the phrasing can be clearer to mention using `list` after a clear to remove any additional filters

Reply:
Because the current impression is that the clear only applies to the filtered list

Reply:
Ok I found something

<img width="634" alt="image" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/0307b7a4-89f4-4058-87e6-eae6edb20fdc">

Add transaction in `ModelManager` resets the predicate, but this doesn't apply for the other commands